plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/animator/CameraAnimator.kt

```diff
@@ -81,6 +81,21 @@ abstract class CameraAnimator<out T> (
    */
   final override fun start() {
     postOnAnimatorThread {
+      if (duration == 0L && startDelay == 0L) {
```

Comment:
Should we limit it to run on API <= 23 only ?

Reply:
refactored the code a bit, please take another look.